lib/route.js

```diff
@@ -99,6 +99,10 @@ function buildRouting (options) {
 
   // Convert shorthand to extended route declaration
   function prepareRoute (method, url, options, handler) {
+    if (url && typeof url !== 'string') {
+      throw new Error(`URL for method:${method} must be a string`)
```

Comment:
I would change it a bit the wording since it is a sync error
```suggestion
      throw new Error(`URL must be a string. Received ${typeof url}`)
```